core/scripts/sql/texera_ddl.sql

```diff
@@ -154,6 +154,7 @@ CREATE TABLE IF NOT EXISTS workflow_executions
     `eid`             INT UNSIGNED AUTO_INCREMENT NOT NULL,
     `wid`             INT UNSIGNED NOT NULL,
     `vid`             INT UNSIGNED NOT NULL,
+    'name'            VARCHAR(128) NOT NULL DEFAULT 'Untitle Execution',
```

Comment:
please merge latest master and avoid adding unrelated changes

Reply:
deleted